README.md

```diff
@@ -1,2 +1,4 @@
 # hello-world
 this is the intro repository for Blake Adams
+
+My name is Blake Adams and I like to pressure wash things, hang out with family friends (socially distanced now), enjoy great meals at local restaurants, and taking my dog to the dog park. I've worked in IT for the past five years, but I've only been on the sales side. I'm eager to learn about the technical side and expand my skills through this great program.
```

Comment:
A cool thread that I found and will be looking into is right here: https://github.com/gerryguy311/Free_CyberSecurity_Professional_Development_Resources.git